networkx/algorithms/bipartite/generators.py

```diff
@@ -551,8 +551,8 @@ def gnmk_random_graph(n, m, k, seed=None, directed=False):
 
     Examples
     --------
-    from nx.algorithms import bipartite
-    G = bipartite.gnmk_random_graph(10,20,50)
+    >>> from nx.algorithms import bipartite
```

Comment:
I think some of these (the ones that just create a random graph and don't do anything else)
are intentionally not using the `>>>` prefix so they still show on the docs but don't get run
through the doctests.  But they do look ugly in the docs -- If we're going to try keeping them
out of the doctests we should make sphinx present them in a quoted environment anyway.

@rossbar do you have ideas about whether these should be in doctests, and if not, how to format them so they look like code?

Reply:
I'm curious why these would be intentionally skipped. Most other graph generators use real examples with doctests. I think it's generally a good idea to execute code. Observe that the original code has a typo that would get caught if run during doctests.

Also, using `# doctest: +SKIP`, while kinda ugly, does skip running it.

Reply:
Yikes — a typo that is not caught is quite bad.  I don’t even see the typo.

And it looks like @rossbar has approved these. So I think my concerns are not valid.  I’ll mark this as resolved.

Reply:
> do you have ideas about whether these should be in doctests, and if not, how to format them so they look like code?

Generally I agree that there isn't a ton of value in these types of examples that just show a function being called without any sort of inspection/use of the output, but improving the docstring examples is a separate, more time-intensive task. I definitely agree with @eriknw that everything that is presented in `Examples` sections should be executed to prevent errors in the docstrings (setting aside `# doctest: +SKIP` for special cases)!